### ISSUE TITLE
Testing for fluent-bit

### ISSUE THREAD

**Issue by guineveresaenger** [samsung-cnct/chart-fluent-bit#7]
Add unit tests for fluent-bit, testing input, output, and filter plugins.

This will involve research on how to set up tests for fluent-bit.

**Comment by davidewatson** [comment]
From the deprecated kraken-logging-fluent-bit-daemonset repo

Currently, the CI run on this repository does not run comprehensive tests.

CI should:

- spin up a lightweight cluster (https://github.com/samsung-cnct/k2cli#how-to-create-a-small-research-or-development-cluster)
- deploy the fluent-bit daemonset on the cluster
- run logs on the pods and verify that they contain docker logs, journal logs, kernel logs, etc.

**Comment by guineveresaenger** [comment]
This ticket is specifically for unit tests, not a CI run. I think that should be a separate issue.
Also, a lightweight cluster may not be large enough to test logging.

**Comment by coffeepac** [comment]
blocked by move to gitlab

helm linting
create chart in fresh namespace in gitylab cluster (kubernetes)

**Comment by guineveresaenger** [comment]
Another possibility for CI tests has an example here, with a Makefile:

https://github.com/samsung-cnct/cyklops-events/pull/7/files

**Comment by mikeln** [comment]
Superceeded by new jenkins conversion
